src/components/Ticketmaster.js

```diff
@@ -242,22 +245,29 @@ const Ticketmaster = function () {
       </SearchGroup>
       <ResultTable>
         <tbody>
-          {data["_embedded"]["events"].map(tmevent => (
-            <tr key={tmevent.id}>
-              <td>{tmevent.name}</td>
-              <td>{createDateLocation(tmevent.dates.start.localDate, tmevent._embedded.venues[0].name)}</td>
-              <td>{createCityState(tmevent)}</td>
-              <td><a href={tmevent.url} rel="noopener noreferrer" target="_blank">Buy Tickets</a></td>
-            </tr>
-          ))}
+          {isResults ?
+            data["_embedded"]["events"].map(tmevent => (
+              <tr key={tmevent.id}>
+                <td>{tmevent.name}</td>
+                <td>{createDateLocation(tmevent.dates.start.localDate, tmevent)}</td>
+                <td>{createCityState(tmevent)}</td>
+                <td><a href={tmevent.url} rel="noopener noreferrer" target="_blank">Buy Tickets</a></td>
+              </tr>
+            )) :
+            <h1>No results found</h1>
```

Comment:
should this be a modal? both?

Reply:
I think modal think both would be nice I will look into having a modal aswell